### ISSUE TITLE
[Feature Request] Support All Blkio Throttles Available in Engine API

### ISSUE THREAD

**Issue by stuartthomson** [docker/cli#3325]
The Docker Engine API supports a number of Blkio throttles that the CLI does not. (See [here](https://docs.docker.com/engine/api/v1.40/#operation/ContainerCreate), under `.HostConfig`)

I would like to be able to use these throttles from the CLI, both in `container create` and `container update`.

**Comment by thaJeztah** [comment]
These options should already be available through the `--device-xxx` options (see https://github.com/moby/moby/pull/15879 and https://github.com/moby/moby/pull/14466);


```
      --device-read-bps list           Limit read rate (bytes per second) from a device (default [])
      --device-read-iops list          Limit read rate (IO per second) from a device (default [])
      --device-write-bps list          Limit write rate (bytes per second) to a device (default [])
      --device-write-iops list         Limit write rate (IO per second) to a device (default [])
```


For `docker container update`, I haven't checked if the API currently supports it (if not, that would need changes in the engine first); if it _does_ support updating these, it would need a design for the CLI; these options can be provided multiple times (per device), so we'd need to find the right UX to allow updating these values (without _removing_ existing values that should not be updated).

Possibly a similar approach as `docker service update` (which has `--xxx-add` `--xxx-remove` options to specify what "diff" to make when updating),

**Comment by stuartthomson** [comment]
Apologies, I didn't check thoroughly enough where these are/are not supported. I was actually trying to update a running container when I created the issue, just skimmed the `create` docs and must have missed it!

The engine [API ](https://docs.docker.com/engine/api/v1.40/#operation/ContainerUpdate )at least implies that these are supported for update too.

**Comment by thaJeztah** [comment]
> The engine API at least implies that these are supported for update too.

Yes, the API describes it, but I was a bit cautious there. I know there's some places in the API where the same types/structs are used for multiple endpoints (e.g. for both create and update), but not all fields are used by the API server. Whenever I encounter such situations, I tend to make a note (or a PR) to split those into separate types, but there's probably still places where this is the case.
_Ideally_ such cases should either "work" or produce an error (if the field is not supported by the endpoint), but given that those PRs were created before `docker container update` existed, I'm not sure if that's verified; also if these cgroups can be added/removed without in `docker update` without side-effects (I know there's some cgroups that cannot be updated/removed (or require special handling) after the container namespace is created); at least it would be good to have test-cases for this; a quick glance didn't show any for these fields in https://github.com/moby/moby.

**Comment by stuartthomson** [comment]
An update: I tried calling the update API manually for this, with:
```bash
$ curl --unix-socket /var/run/docker.sock -H "Content-Type: application/json" \
  -d '{"BlkioDeviceReadBps": [{"Path": "/path/to/device", "Rate": 1}]}' \
  -X POST http://localhost/v1.41/containers/<container_id>/update
```
In response I got simply `{"Warnings":null}`, and inspecting the container shows `"BlkioDeviceReadBps": null`, so it seems like it didn't take.
